stackslib/src/chainstate/stacks/block.rs

```diff
@@ -40,6 +41,15 @@ use crate::chainstate::stacks::{Error, StacksBlockHeader, StacksMicroblockHeader
 use crate::core::*;
 use crate::net::Error as net_error;
 
+/// Quietable error
+macro_rules! qerror {
```

Comment:
Please drop this macro altogether.  If something is `error!`-level, it should never be quieted.